src/plugins/intel_cpu/tests/functional/single_layer_tests/gather.cpp

```diff
@@ -274,6 +274,27 @@ INSTANTIATE_TEST_SUITE_P(smoke_static_1D,
                                             ::testing::Values(additionalConfig[0])),
                          GatherLayerTestCPU::getTestCaseName);
 
+const std::vector<std::vector<ov::test::InputShape>> staticInputShapes1DI32 = {
+    {{{}, {{1}}}, {{}, {{1}}}},   {{{}, {{2}}}, {{}, {{2}}}},   {{{}, {{3}}}, {{}, {{3}}}},
```

Comment:
Don't think we need all these instances. Lets leave just 2 or 3.

Reply:
OK, reduced number of test cases to 3